src/routes/settings.tsx

```diff
@@ -49,7 +51,17 @@ function SettingsIndexRoute() {
   const [selectedColorInput, setSelectedColor] = useState("");
   const [selectedLogoInput, setSelectedLogo] = useState("");
   const [colorPickerOpen, setColorPickerOpen] = useState(false);
-  const debounceTimeoutRef = useRef<NodeJS.Timeout | null>(null);
+  const debounceTimeoutRef = useRef<ReturnType<typeof setTimeout> | null>(null);
+
+  const themeColorDefaultOptions = useMemo(
+    () => [
+      { name: "Lipsanen Lipa-Betoni Oy", value: theme.palette.companyDefault.lipsanenLipsaBetoniOy },
+      { name: "Rakennus Ahola", value: theme.palette.companyDefault.rakennusAhola },
+      { name: "Moduls", value: theme.palette.companyDefault.moduls },
+      { name: "HB-Porras", value: theme.palette.companyDefault.hbPorras },
```

Comment:
The names could be localized

Reply:
Are there multi-language versions of company names? I would assume they are the registered names of the companies, but I might also be incorrect 🤔

Reply:
None that I have seen somewhere. So it is fine as is 👍